spinn_front_end_common/interface/abstract_spinnaker_base.py

```diff
@@ -1424,8 +1447,9 @@ def generate_file_machine(self):
         }
         outputs = ["FileMachine"]
         executor = PACMANAlgorithmExecutor(
-            algorithms=[], optional_algorithms=[], inputs=inputs,
-            xml_paths=self._xml_paths, required_outputs=outputs,
+            algorithms=[], optional_algorithms=[], inputs=inputs, tokens=[],
```

Comment:
missing parameter. code wont run as it currently stands. bounced till fixed

Reply:
Fixed